tests/helpers_tests/test_condition.py

```diff
@@ -108,6 +122,214 @@ def test_evaluate__returns_null__when_condition_has_an_invalid_type_property(sel
 
         self.assertIsNone(evaluator.evaluate(0))
 
+    def test_evaluate__returns_true__when_user_version_matches_target_version(self):
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
+            semver_equal_2_0_condition_list, {'Android': '2.0'}, self.mock_client_logger
+        )
+
+        self.assertStrictTrue(evaluator.evaluate(0))
+
+        evaluator = condition_helper.CustomAttributeConditionEvaluator(
```

Comment:
Separate each case into separate definition. In case of failure It would be helpful to identify the exact failed scenario.